templates/repo/issue/view_content.tmpl

```diff
@@ -145,6 +145,7 @@
 						</div>
 					</div>
 				{{else if eq .Type 7}}
+					{{if .Label}}
 					<div class="event">
```

Comment:
Yes, I understand why you're using two `if`s, I'm saying that it should be like this
```
{{else if eq .Type 7}}
	{{if .Label}}
		<div class="event">
			...
```

instead of like this

```
{{else if eq .Type 7}}
	{{if .Label}}
	<div class="event">
		...
```